Podfile.lock

```diff
@@ -450,7 +450,7 @@ PODS:
     - WordPressKit (~> 4.18-beta)
     - WordPressShared (~> 1.12-beta)
     - WordPressUI (~> 1.7-beta)
-  - WordPressKit (4.38.0-beta.1):
+  - WordPressKit (4.38.0-beta.2):
```

Comment:
The release branch had to introduce an hotfix version of WordPressKit. The fix from that version needs to be brought back into `develop`, otherwise the changes from this branch to fix Reader comments won't compile.

Reply:
Doesn't need to be fixed this time, but we shouldn't use beta pods in production.

In this case it would've been preferable to cut a `4.38.1` release containing just the changes 🙂